tests/client/test_redirects.py

```diff
@@ -242,23 +241,6 @@ class MockDispatch(AsyncDispatcher):
             response = await response.anext()
 
 
-@pytest.mark.usefixtures("async_environment")
-async def test_redirect_loop():
-    client = AsyncClient(dispatch=MockDispatch())
-    with pytest.raises(RedirectLoop):
-        await client.get("https://example.org/redirect_loop")
```

Comment:
I guess we *could* keep this test in, and just ensure that `TooManyRedirects` is raised?

Reply:
There is already test for TooManyRedirects with 21 redirects so i thought it's gonna be enough but sure more is better.